lib/sidebar.rb

```diff
@@ -29,185 +29,193 @@ def pre_render(page)
     end
 
     def post_write(site)
+      files = parse_files(site)
+      write_sidebar_to_files(files)
+    end
+
+    private
+
+    def parse_files(site)
```

Comment:
Codacy found an issue: [Assignment Branch Condition size for parse_files is too high. [<8, 17, 1> 18.81/15]](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=6198407)